packages/shared/src/cli/git.tag.ts

```diff
@@ -0,0 +1,6 @@
+import { execFileSync } from 'child_process';
+
+export const GitTag = {
```

Comment:
want to make this a function so we dont execute files every time this script is imported, this allows us to export it inside of the main index, and then you can import it from the index rather than via /build

```
export function GitTag() { return ... }
```

Reply:
But it is a sync command so executing it every time is not nice and we don't want it in the index because it may not be valid once deployed

Reply:
exactly you are waiting on two sub processes to run even if you dont even use these values.

the infra version info already has a caching mechanism, 

The cli in production will never want it never to run, and then adds a dep on git for our container.

Reply:
You are only waiting on this if you *do* use this process otherwise the module would not import the version module or this module directly